scripts/apps/contacts/components/Form/ProfileDetail.tsx

```diff
@@ -129,17 +171,43 @@ export class ProfileDetail extends React.Component<any, any> {
                     </div>
                 }
 
-                {!readOnly &&
-                    <Row>
+                <Row flex={true}>
+                    {get(this.state, 'contactTypes.length', 0) > 0 && (
```

Comment:
Use optional chaining and nullish coalescing instead of `lodash.get` to preserve type safety: `this.state?.contactTypes?.length ?? 0 > 0`

Reply:
This PR has to be merged through to v1.30 of Superdesk which uses Typescript 3.0, where optional chaining and nullish coalescing is not available.